### DISCUSSION TITLE
Importar planes desde Banner

### DISCUSSION THREAD

**Discussion by wareval0** [Open-Source-Uniandes/Mi-Horario-Uniandes#61] [Ideas]
Sería genial, después de crear mi horario perfecto, poder importar el plan de horario en Banner para evitar ingresar manualmente curso por curso, ya que es un proceso bastante demorado y puede producir muchos errores. Además, en momentos críticos (cuando un plan no sirve por cupos llenos), es fundamental la velocidad a la hora de inscribir horario; qué mejor que _**Mi-Horario-Uniandes**_ pueda generar el mejor horario disponible en ese momento y de una vez poder importarlo para inscribir antes de que se llenen los cupos.

**Comment by c4ts0up** [comment]
Creo que ahí serían dos cosas, no? Primero, poder exportar el plan de Mi Horario a un formato y, segundo, poder importar el plan exportado en Banner. Siento que esto segundo nos limita porque es una herramienta sobre la cual no tenemos mucho control. Pero la primera parte la podemos ir haciendo y está considerada en las HU. Teníamos en mente algo como exportar el plan a un archivo .txt con los códigos NRC de las materias, los cuales se pueden ingresar fácil y rápidamente durante inscripción.
